packages/client/modules/meeting/components/ScaleDropdownMenuItem.tsx

```diff
@@ -61,42 +63,50 @@ const ScaleActionButtonGroup = styled('div')({
   marginBottom: 'auto'
 })
 
-const ScaleDropdownMenuItem = forwardRef((props: Props, ref) => {
-  const {scale, dimension, menuProps} = props
-  const {closePortal} = menuProps
-  const {values} = scale
+const StarterIcon = styled(Icon)({
+  fontSize: 18,
+  lineHeight: '24px',
+  paddingLeft: 4
+})
 
+const ScaleDropdownMenuItem = forwardRef((props: Props, ref) => {
+  const {scale, dimension, closePortal, scaleCount} = props
+  const {id: scaleId, isStarter, name: scaleName, values} = scale
+  const {id: dimensionId, team} = dimension
+  const {id: teamId} = team
   const atmosphere = useAtmosphere()
   const {submitMutation, submitting, onError, onCompleted} = useMutationProps()
 
   const setScale = (scaleId: any) => () => {
     if (submitting) return
     submitMutation()
-    UpdatePokerTemplateDimensionScaleMutation(atmosphere, {dimensionId: dimension.id, scaleId}, {onError, onCompleted})
+    UpdatePokerTemplateDimensionScaleMutation(atmosphere, {dimensionId, scaleId}, {onError, onCompleted})
     closePortal()
   }
 
   return (
     <MenuItem
       ref={ref}
-      key={scale.id}
-      onClick={setScale(scale.id)}
+      onClick={setScale(scaleId)}
       label={
         <ScaleDetails>
           <ScaleNameAndValues >
-            <ScaleName>{scale.name}</ScaleName>
+            <ScaleName>
+              {scaleName}
+              {isStarter && <StarterIcon>public</StarterIcon>}
+            </ScaleName>
             <ScaleValues>
               {
                 [...values.filter(({isSpecial}) => !isSpecial).map(({label}) => label), PokerCards.QUESTION_CARD, PokerCards.PASS_CARD].join(", ")
               }
             </ScaleValues>
           </ScaleNameAndValues>
-          <ScaleActionButtonGroup key={`scale_edit_${scale.id}`} >
+          <ScaleActionButtonGroup>
             <ScaleActions
-              scaleId={scale.id}
-              isStarter={scale.isStarter}
-              scaleCount={0}
```

Comment:
these things are always hard to catch on big PRs.
typically, what i like to do is:

```js
// FIXME
const scaleCount = 0
...
<ScaleActions scaleCount={scaleCount}/>
```

by declaring the variable up above & putting a fixme on it it makes it a little easier to spot before committing, but there's no perfect method